src/notation/qml/MuseScore/NotationScene/internal/CustomiseControlPanel.qml

```diff
@@ -72,7 +72,6 @@ RowLayout {
     }
 
     FlatButton {
-        Layout.alignment: Qt.AlignRight
```

Comment:
@cbjeukendrup 
I don't see a problem with this line. Could you explain?

Reply:
Hm, I don't really remember exactly what was the problem, since I wrote this code years ago by now. I'd have to recheck, which I can hopefully do when I finish rebasing my qt6 branch. I do remember that it's one of those cases where an innocent line causes things to break weirdly with Qt 6 where Qt 5 was fine.

Reply:
I suggest merge in any case, as far as I can see, this change does not make things worse.
@Eism Do you agree?

Reply:
ok, agree